.github/workflows/integ_test_logging.yml

```diff
@@ -39,7 +39,6 @@ jobs:
       scheme: ${{ matrix.platform == 'watchOS' && 'AWSCloudWatchLoggingPluginIntegrationTestsWatch' || 'AWSCloudWatchLoggingPluginIntegrationTests' }}
       platform: ${{ matrix.platform }}
       project_path: ./AmplifyPlugins/Logging/Tests/AWSCloudWatchLoggingPluginHostApp
-      xcode_version: ${{ matrix.platform == 'watchOS' && '15.0' || '14.3' }}
```

Comment:
These tests used to run on Xcode 15.0 for watchOS, but now they will use 14.3.
Have you checked it still works?

Reply:
good catch, will update it.